### DISCUSSION TITLE
Is Lua faster than .NET

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/runtime#88439] [Q&A]
Lua is faster compared to python, I heard. Anyone knows

**Comment by danmoseley** [comment]
What do you mean by faster - at what job, and by what measures?

**Reply by danmoseley** [reply]
Having said that, on these benchmarks C# is generally similar or faster than Java, and Java is apparently much faster than Lua:

https://benchmarksgame-team.pages.debian.net/benchmarksgame/fastest/lua-java.html

But relevance to real world is questionable. What are you trying to achieve?

**Comment by ShalokShalom** [comment]
> Lua is faster compared to python, I heard. Anyone knows

1) Pretty much any language is faster than Python 

2) Languages itself are not any fast at all, to be precise. Implementations are. 

LuaJIT as an example is much faster than Lua. CPython is slower than PyPy and Cython. 

3) Performance never happens without context. How fast: When? For whom? How long?
